oviewer/convert_es_test.go

```diff
@@ -343,12 +340,120 @@ func Test_parseCSI(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := parseCSI(tt.args.params)
-			if (err != nil) != tt.wantErr {
-				t.Errorf("parseCSI() error = %v, wantErr %v", err, tt.wantErr)
+			if got := parseSGR(tt.args.params); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("parseSGI() = %v, want %v", got, tt.want)
 			}
-			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("parseCSI() = %v, want %v", got, tt.want)
+		})
+	}
+}
+
+func Test_parseSGR2(t *testing.T) {
```

Comment:
Why 2 ?

You could simply add two t.Run block in Test_parseSGR

Reply:
I want to separate them because there are too many, but it is hard to think of names...